site/src/sphinx/advanced-structured-logging.rst

```diff
@@ -157,3 +151,27 @@ automatically for you:
             ...
         }
     }
+
+.. _nested-log:
+
+Nested log
+----------
+
+You can set the logs from other RequestLog_:
```

Comment:
Could you add a paragraph about the purpose of this feature? e.g. Sometimes a request is consisted of more than one child request. ...